docs/config.md

```diff
@@ -126,6 +126,7 @@ due to lack of stable semantic convention.
 
 | ID                    | Instrumented library                                                                                                                                                                            | Supported versions | Instrumentation type    | Status                                                                                                                            |
 |-----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------|-------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
+| `AZURE`               | Azure SDK                                                                                                                                                                                       | *                  | source                  | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
```

Comment:
Please add link to the Azure SDK library as for Elastic search and other records. ASPNET and ASPNETCORE is kind of exception,

Reply:
Added a link.